src/utils/hashPass.js

```diff
@@ -0,0 +1,44 @@
+const bcrypt = require('bcrypt');
+const { saltRounds } = require('../config/index');
```

Comment:
You don't need to import index. Try it out `const { saltRounds } = require('../config');`